ci/requirements-3.6.run

```diff
@@ -14,7 +14,7 @@ html5lib
 jinja2
 sqlalchemy
 pymysql
-feather-format
+# feather-format (not available on defaults ATM)
```

Comment:
u need a separate file (and a different job tag)
also ok to just have  3 circle  jobs